app/models/smart_proxy.rb

```diff
@@ -88,22 +88,16 @@ def has_feature?(feature)
     self.features.any? { |proxy_feature| proxy_feature.name == feature }
   end
 
-  def version
-    result = {}
-    begin
-      Timeout::timeout(20) do
-        version = ProxyAPI::Version.new(:url => url).version
-        result[:success] = true
-        result[:message] = version
-      end
-    rescue *HTTP_ERRORS => exception
-      raise ::Foreman::WrappedException.new exception, N_("Unable to connect to smart proxy")
-    end
-    result
+  def features_to_lowercase
+    features.pluck("LOWER(REPLACE(name, ' ', '_'))").uniq
```

Comment:
If this is strictly to load partials & it has everything to do with the presentation layer, shouldn't this be in the helper?